src/Pages/Reset.js

```diff
@@ -1,13 +1,13 @@
-import React, { useEffect, useState } from "react";
-import { useAuthState } from "react-firebase-hooks/auth";
-import { useNavigate } from "react-router-dom";
-import { Link } from "react-router-dom";
-import { auth, sendPasswordReset } from "../Components/firebase";
-import "./Reset.css"
+import React, { useState } from 'react';
+// import { useAuthState } from 'react-firebase-hooks/auth';
```

Comment:
removed unnecessary imports and commented out functionality that will be applied later